src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -420,9 +418,7 @@ public void addProperty(String propertyName, Object propertyValue) throws SFExce
           break;
 
         case PRIVATE_KEY_FILE_PWD:
-          if (propertyValue != null) {
-            privateKeyPassword = (String) propertyValue;
-          }
+          if (propertyValue != null) {}
```

Comment:
can this be removed?

Reply:
wait, why this can be removed?

Reply:
It is doing nothing in the new code.

Reply:
Oops, took this change back out. The privateKeyPassword appears to never get used or passed along in SFSession, though.